text/0043-phrase-query.md

```diff
@@ -0,0 +1,172 @@
+- Title: Phrase Query
+- Start Date: 2021-06-02
+- Specification PR: [#43](https://github.com/meilisearch/specifications/pull/43)
+- MeiliSearch Tracking-Issues:
+
+# Phrase Query
+
+## 1. Feature Description and Interaction
+
+### I. Summary
+
+MeiliSearch does not allow users a way to write a strict query in order to ask the engine to be more strict in its selection of candidates for search results. The Phrase Query feature adds a simple syntax available to users to require the engine to select documents that strictly match some phrase, indicated by quotation marks. That is, without typography, n-gram, wordsplit, prefix, synonym, and, proximity. In addition, the expression of a Phrase Query is case insensitive.
+
+### II. Motivation
+
+This feature is driven by user needs. Indeed, let's take an example recently brought up in our community Slack.
+
+The user in question would like to be able to retrieve specifically the document containing the unique ISBN identifier and only that one. In a UX context of type as you search, this is impossible today without impacting the UI/UX or finding a workaround.
+
+- It could have had a specific search field only dedicated to this ISBN field in order to make it a filter and using the `=` operator. Not great for the UX.
+
+- It could also have kept a single search field and detected that an ISBN was entered in the search field using a regex to inject that filter at that point. Not great for the developper experience, moreover, what happens when a pattern cannot be specifically determined?
+
+The Phrase Query feature will easily solve this case but will also adapt to the needs of the user performing the search.
```

Comment:
When I read the user need here, It seems that the proposed solution is not the best way to do it. What if we asked the user to put his ISBN as `primaryKey` and to retrieve his document using the [get documents](https://docs.meilisearch.com/reference/api/documents.html#documents) route.

I think this feature goes further than that. It serves many other purposes as well. As you said above, it allows you to ignore the ranking rules for words between `"`. So it could allow, like google, to search `a green "t-shirt"` and to have at least documents with `t-shirt`. This word will not be removed by the ranking rule `Word`.

Reply:
Using the getDocuments route is a solution that will only work if the need is limited to searching for an ISBN value. If the search bar should also allow searching within the document attributes, we come back to the workaround to detect the ISBN pattern and make a filter of it or make a request on getDocuments route.

Totally agree with the second point. The Phrase Query allows for more than just the specific case cited.